app/helpers/articles_helper.rb

```diff
@@ -47,8 +52,8 @@ def should_show_crossposted_on?(article)
   end
 
   def get_host_without_www(url)
-    url = "http://#{url}" if URI.parse(url).scheme.nil?
-    host = URI.parse(url).host.downcase
+    url = "http://#{url}" if Addressable::URI.parse(url).scheme.nil?
```

Comment:
I have no problem with it, but why this change that is not I18n related?

Reply:
In some sense it does. I believe they are here after I found they didn't handle non-ASCII path very well.